cpp/community_detection_module/algorithm/louvain.cpp

```diff
@@ -68,12 +68,12 @@ void LoadUndirectedEdges(const mg_graph::GraphView<> &memgraph_graph, GrappoloGr
     edge_index++;
   }
 
-  auto edge_list_ptrs = new long[number_of_vertices + 1];
+  auto edge_list_ptrs = static_cast<long *>(malloc((number_of_vertices + 1) * sizeof(long)));
```

Comment:
community detection library uses malloc and free instead of new + delete

Reply:
Where do we free this if we allocated with malloc? Shouldn't we also have `free`

Reply:
It's freed in the louvain (community detection) library, this should've been crashing long ago but didn't.